### PR TITLE
Added in manifest info

### PR DESCRIPTION
**Pull request by Yash-Singh1**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any

# Data

Look at the commit messages for sources

### CODE REVIEW

**Comment by ddbeck** [comment]
Hi @Yash-Singh1 and thanks for your patience while we got around to reviewing this PR.

Looking this over, I don't think route to a PR is going to work. For many of these features, it looks like the caniuse data is reporting that the current version is supported, not a true version added value (e.g., some of the features that are version 87 in your PR are now shown as 88). This data also appears to be intermingled with BCD's own data.

To move these details forward, I think we need some independent source of information about the features' support, such as Chrome bugs or commits or direct tests. If you'd like to work on updating these values and citing sources for them in this PR, then we can leave this open while you work on that, or we can close it until another opportunity arises.

Thank you!

**Comment by Yash-Singh1** [comment]
@ddbeck I just updated the PR. I wasn’t able to find samsung and webkit data though, so I just removed those.

File: html/manifest/scope.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
           "spec_url": "https://w3c.github.io/manifest/#scope-member",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": "53"
             },
             "chrome_android": {
               "version_added": null
```
Comment (ddbeck) [inline html/manifest/scope.json:13]: Is there a reason this was set only for Chrome and not Chrome for Android? This is different from the others in this PR.

**Review by ddbeck — APPROVED** [review]
Thank for filling in all the details via the commit messages, @Yash-Singh1! Looks good. 🎉

**Review by ddbeck — COMMENTED** [review]
Oh wait, I forgot I had one question about the scope data. 🤦  Just need to answer that before we go ahead with merging this.

File: html/manifest/scope.json (hunk 1 above)
Comment (Yash-Singh1) [reply html/manifest/scope.json:13]: I think I just forgot to add it in

Comment (Yash-Singh1) [reply html/manifest/scope.json]: 91851d8

**Review by ddbeck — APPROVED** [review]
This is a really well-developed update. Thank you! 🎉

**Pull request merged**